### PR TITLE
command/agent: allow the job GC interval to be configured

### PR DESCRIPTION
**Pull request by pete-woods**
This avoids having to schedule a service to call the GC endpoint more frequently.

### CODE REVIEW

**Comment by pete-woods** [comment]
Hi @notnoop, @cgbaker, @jazzyfresh ! Would any of you folks have a moment to have a look at this (hopefully) straightforward PR?

Sorry for the random `@`, you all just seemed to be involved in many of the recent PRs. :)

**Comment by pete-woods** [comment]
Thanks @langmartin !

**Comment by pete-woods** [comment]
Getting this upstreamed would be very helpful to us here at CircleCI!

File: website/source/docs/configuration/server.html.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ server {
   terminal state before it is garbage collected and purged from the system. This
   is specified using a label suffix like "30s" or "1h".
 
+- `job_gc_interval` `(string: "5m")` - Specifies the interval between the job
+  garbage collection loops. This is specified using a label suffix like "30s"
+  or "3m".
```
Comment (schmichael) [inline website/source/docs/configuration/server.html.md]: It would be nice to describe _why_ someone might want to change this parameter as it's a far more niche tunable than `job_gc_threshold`.

Perhaps something like:

```
- `job_gc_interval` `(string: "5m")` - Specifies the interval between the job
  garbage collections. Only jobs who have been terminal for at least
  `job_gc_threshold` will be collected. Lowering the interval will perform more
  frequent but smaller collections. Raising the interval will perform collections
  less frequently but collect more jobs at a time. This is specified using a
  label suffix like "30s" or "3m".
```

But that's a bit wordy, so I don't know. Not a blocker, but if you or anyone else wants to sneak a little more detail into the docs I think that would be helpful.

Comment (pete-woods) [reply website/source/docs/configuration/server.html.md]: I've rebased and added some extra info.

Comment (schmichael) [reply website/source/docs/configuration/server.html.md]: Perfect! Thanks! 0.9.4 final should go out next week and then we'll merge.

**Comment by schmichael** [comment]
Thanks again @pete-woods ! You may have had the first merge of 0.10.0. :smile:

**Comment by pete-woods** [comment]
Thanks, everyone. I've got another one (#6003) that adds some new metrics we use for monitoring our nomad clusters here at CircleCI, if any of you have time for more reviews. 🙂
